tests/spiders.py

```diff
@@ -255,7 +255,7 @@ class CrawlSpiderWithParseMethod(MockServerSpider, CrawlSpider):
     A CrawlSpider which overrides the 'parse' method
     """
     name = 'crawl_spider_with_parse_method'
-    custom_settings = {
+    custom_settings: dict = {
```

Comment:
I don't really know why this is necessary, since `custom_settings` is annotated in the base class :shrug: